appveyor.yml

```diff
@@ -6,15 +6,22 @@ branches:
 
 cache:
   # Note that we don't use cache for local Maven repository - see https://github.com/SonarSource/sonar-java/pull/525
-  - C:\ProgramData\chocolatey\bin -> appveyor.yml
-  - C:\ProgramData\chocolatey\lib -> appveyor.yml
-  - C:\bin\apache-maven-3.2.5 -> appveyor.yml
+  - C:\Users\appveyor\maven -> appveyor.yml
```

Comment:
@Godin is this worth doing in the first place? is Apache download servers are fast & stable, then probably not. Perhaps this was initially introduced exactly because of Chocolatey's own instability.